### PR TITLE
update rvm ruby and bundle

### PR DESCRIPTION
**Pull request by jmartin-r7**
Looks like these updates are fairly overdue.

### CODE REVIEW

**Comment by adfoster-r7** [comment]
This now builds successfully for me 🎉 

image

Looks like the top tabs are different on the live resources.metasploit.com site though:

image

File: metasploit-resource-portal/source/_topnav.erb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 <% nav as: :pills, position: :static do %>
-  <%= link_to 'Docs',      'https://help.metasploit.com' %>
-  <%= link_to 'API',       'https://rapid7.github.io/metasploit-framework/api/' %>
-  <%= link_to 'Community', 'https://community.rapid7.com/community/metasploit' %>
-  <%= link_to 'Support',   'https://www.rapid7.com/support/' %>
-  <%= link_to 'Stats',     'pull_requests.html' %>
+  <li><a href="https://help.metasploit.com">Docs</a></li>
```
Comment (adfoster-r7) [inline metasploit-resource-portal/source/_topnav.erb:2]: Existing issue, looks like this cert has recently expired

image

I think it's meant to be `https://docs.rapid7.com/metasploit/`

File: metasploit-resource-portal/source/_topnav.erb [hunk 2]
```diff
@@ -1,9 +1,8 @@
 <% nav as: :pills, position: :static do %>
-  <%= link_to 'Docs',      'https://help.metasploit.com' %>
-  <%= link_to 'API',       'https://rapid7.github.io/metasploit-framework/api/' %>
-  <%= link_to 'Community', 'https://community.rapid7.com/community/metasploit' %>
```
Comment (adfoster-r7) [inline metasploit-resource-portal/source/_topnav.erb:4]: Just wanted to confirm this community link was intentionally removed before landing :+1:

Current production:

image

Comment (jmartin-r7) [reply metasploit-resource-portal/source/_topnav.erb:4]: Yes that site was removed a few years ago.

**Pull request merged**